src/trivia/libraries/problems/problems.go

```diff
@@ -2,17 +2,14 @@ package problems
 
 import (
 	"encoding/csv"
-	"flag"
+	"fmt"
 	"os"
 	"strconv"
 	"strings"
 )
 
 // GetProblems returns trivia questions
-func GetProblems() ([]Problem, error) {
-	csvFilename := flag.String("csv", "problems.csv", "a csv file in the format of 'question,option,option,option,answer'")
-	flag.Parse()
-
+func GetProblems(csvFilename *string) ([]Problem, error) {
```

Comment:
nit - this should just pass a string

in general you only pass *object to things where you expect them to modify the object which this would not. Passing a copy makes it so they can't modify the source object even if they want to.

Reply:
848c0f5